config/config.proto

```diff
@@ -54,6 +54,10 @@ message HipChatConfig {
   optional bool notify = 4 [default = false];
   // Notify when resolved.
   optional bool send_resolved = 6 [default = false];
+  // Prefix to be put in front of the message (useful for @mentions, etc.)
+  optional string prefix = 7 [default = ""];
+  // Format the message as "html" or "text"
+  optional string message_format = 8 [default = "html"];
```

Comment:
This would need a check in `Config.Validate()` (https://github.com/prometheus/alertmanager/blob/master/config/config.go#L43) to make sure it's set to a valid value.

But I think it may be better to use an enum (and then AFAIK it wouldn't need extra validation either):

```
enum MessageFormat {
  HTML = 0;
  TEXT = 1;
}
optional MessageFormat message_format = 8 [default = HTML];
```